test/unit/spec/MfaVerify_spec.js

```diff
@@ -174,6 +174,39 @@ function (Okta,
       });
     }
 
+    function setupWebauthnOnly() {
+      var setNextResponse = Util.mockAjax();
+      var baseUrl = 'https://foo.com';
+      var authClient = new OktaAuth({url: baseUrl, transformErrorXHR: LoginUtil.transformErrorXHR});
+      var successSpy = jasmine.createSpy('success');
+      var router = new Router(_.extend({
+        el: $sandbox,
+        baseUrl: baseUrl,
+        authClient: authClient,
+        globalSuccessFn: successSpy
+      }));
+      Util.registerRouter(router);
+      Util.mockRouterNavigate(router);
+      setNextResponse([resRequiredWebauthn, resChallengeWebauthn, resSuccess]);
+      router.refreshAuthState('dummy-token');
+      var $forms = $sandbox.find('.o-form');
+      var forms = _.map($forms, function (form) {
+        return new MfaVerifyForm($(form));
+      });
+      if (forms.length === 1) {
+        forms = forms[0];
```

Comment:
can you tell me why we need this? This setup function is only used in one test, so we probably know if this form is present or not.